config.toml

```diff
@@ -31,9 +31,16 @@ api_key_path = "SatSale_API_key"
 # and a tor proxy, default 127.0.0.1:9050 (for Tor Browser use "127.0.0.1:9150")
 #tor_proxy = None
 
-#### Payment method ####
+##### Payment methods - Onchain & Lightning #####
+## Bitcoind node
 pay_method = "bitcoind"
 
+## Pseudo node
+# Switch payment_method to pseudonode if you want to derive addresses from a zpub instead of connecting to a node.
+# And enter your zpub. Derivation path is /0/0, xpubs untested.
+#pay_method = "pseudonode"
+#zpub = ""
```

Comment:
zpub is not so standard thing, maybe better call it xpub and allow xpub / ypub / zpub and do conversion as needed? For example, JoinMarket and other wallets will show you xpub's for segwit wallets. Also, see my other review comment about derivation paths. For all situations would need additional settings to specify address type (you are kinda not forced to use P2WPKH with SatSale) and derivation path. This could be solved with one setting if using output descriptors, but it looks btclib does not support them yet (instead of zpub and potentially other config settings your would then just use `wpkh(xpub...)/84'/0'/0'/0/*`).